Client/Frontend/Browser/SearchEngines.swift

```diff
@@ -176,7 +176,7 @@ class SearchEngines {
     /// Get all bundled (not custom) search engines, with the default search engine first,
     /// but the others in no particular order.
     class func getUnorderedBundledEnginesFor(locale: Locale) -> [OpenSearchEngine] {
-        let languageIdentifier = locale.identifier
+        let languageIdentifier = Locale.preferredLanguages.first!
```

Comment:
`Locale.preferredLanguages.first ?? locale.identifier`

Reply:
I'm fine with that, but preferredLanguages should always have at least one value.

Reply:
`Locale.preferredLanguages.first ?? "en_US"` should work too